text-show-instances.cabal

```diff
@@ -35,6 +35,8 @@ description:         @text-show-instances@ is a supplemental library to @text-sh
                      .
                      * @<http://hackage.haskell.org/package/terminfo             terminfo>@
                      .
+                     * @<https://hackage.haskell.org/package/text-short-0.1.1    text-short>@
```

Comment:
Let's link to https://hackage.haskell.org/package/text-short, not https://hackage.haskell.org/package/text-short-0.1.1.